PROTOCOL.md

```diff
@@ -2334,17 +2411,71 @@ content-type: application/x-ndjson; charset=utf-8
 ### Timestamp Format
 Accepted timestamp format by a delta sharing server: in the ISO8601 format, in the UTC timezone, such as `2022-01-01T00:00:00Z`.   
 
-## API Response Format
+## Delta Sharing Capabilities Header
+This section explains the details of delta sharing capabilities header, which was introduced to help 
+delta sharing catch up with features in [delta protocol](https://github.com/delta-io/delta/blob/master/PROTOCOL.md).
+
+The key of the header is **delta-sharing-capabilities**, the value is semicolon separated capabilities. 
+Each capability is in the format of "key=value1,value2", values are separated by commas.
+Example: "responseformat=delta;readerfeatures=deletionvectors,columnmapping". All keys and values should
+be case-insensitive when processed by the server.
+
+This header can be used in the request for [Query Table Metadata](#query-table-metadata), 
+[Query Table](#read-data-from-a-table), and [Query Table Changes](#read-change-data-feed-from-a-table).
+
+**Compatibility**
+
+<table>
+<tr>
+<th>Client/Server</th>
+<th>Server that doesn't recognize the header</th>
+<th>Server that recognizes the header</th>
+</tr>
+<tr>
+<th>Client that doesn't recognize the header</th>
+<td>Response is in parquet format</td>
+<td>Response should be in parquet format.</td>
+</tr>
+<tr>
+<th>Client that recognizes the header</th>
+<td>The header is ignored at the server, and the format of the response should always be parquet.
+</td>
+<td>The header is processed properly by the server and the response is in the requested format.</td>
```

Comment:
Consider modifying:
The server may choose to respond in parquet format if the table is does not have any advanced features. The server must respond in delta format if the table has advanced features which are not compatible with the parquet format.